source/speech/__init__.py

```diff
@@ -1019,13 +1076,22 @@ def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=cont
 
 	#Finally get speech text for any fields left in new controlFieldStack that are common with the old controlFieldStack (for closing), if extra detail is not requested
 	if autoLanguageSwitching and lastLanguage is not None:
-		speechSequence.append(LangChangeCommand(None))
+		speechSequence.append(
+			LangChangeCommand(None)

```

Comment:
I don't think this should be None!

Reply:
I wasn't so sure, the commented out line above made me question this: `self.lang=lang # if lang else languageHandler.getLanguage()` . Looking into it further 
`command.isDefault` is only used in `ParamChangeTracker.update`, and here it removes any existing `LangChangeCommand` from `_commands`. What isn't clear to me is if there is state stored elsewhere. Does a subsequent command need to be issued to return the language to the system default?

I think generally this isn't something to change in this PR, but it made me wonder.